enzyme/Enzyme/TypeAnalysis/TypeAnalysis.cpp

```diff
@@ -3861,6 +3861,18 @@ void TypeAnalyzer::visitIntrinsicInst(llvm::IntrinsicInst &I) {
         &I);
     return;
 
+  case Intrinsic::smax:
```

Comment:
You'll also need to version gate this, as smax/etc didn't exist in older LLVM's

Reply:
Do you know what version are we talking about? 16 maybe?

Reply:
Not sure, you can check the CI logs, Enzyme-level tests do 11-16, integration runs through 17, and bazel/mlir does nightly-ish

Reply:
https://github.com/llvm/llvm-project/commit/deb4bb2b3abd216aceaa109a5304f025a234540f